installers/pgo-installer/ansible/roles/pgo-installer/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+- name: Set output directory fact
+  set_fact:
+    output_dir: "{{ ansible_env.HOME }}/.pgo/{{ pgo_operator_namespace }}/output"
+  tags: 
+    - always
+
+- name: Ensure output directory exists
+  file:
+    path: "{{ output_dir }}"
+    state: directory
+    mode: 0700
+  tags:
+    - always
+
+- include_vars: openshift.yml
+  when: use_oc | default(false) | bool
+  tags:
+    - always
+
+- name: Use kubectl or oc
+  set_fact:
+    kubectl_or_oc: "{{ openshift_oc_bin if openshift_oc_bin is defined else 'kubectl' }}"
+  tags:
+    - always
+
+- name: Setup Install Container
+  tags:
+    - always
+  block:
+  - name: Create Install Namespace
+    shell: "{{ kubectl_or_oc }} create namespace {{ pgo_installer_namespace }}"
+    ignore_errors: yes
+      
+  - name: Create PGO-Install Service Account
+    shell: "{{ kubectl_or_oc }} create serviceaccount {{ pgo_install_sa }} -n {{ pgo_installer_namespace }}"
+    ignore_errors: yes
+
+  - name: Create PGO-Installer Cluster Role Binding
+    shell: "{{ kubectl_or_oc }} create clusterrolebinding {{ pgo_install_crb }} -n {{ pgo_installer_namespace }} --clusterrole=cluster-admin --serviceaccount={{ pgo_installer_namespace }}:{{ pgo_install_sa }}"
+    ignore_errors: yes
```

Comment:
I think we have to see if we can eliminate the `clusterrolebinding` for the install. That may prevent people from using it, or at least have a way to accomplish the non-multinamespace steps without a clusterrole binding.

I realize this may be a big ask for the first iteration, but wanted to mention it

Reply:
AIUI cluster-admin is required to install. I can look at how to get around it but that might be a design question that would require some discussion.